f5_sphinx_theme/static/css/f5-theme.css

```diff
@@ -54,17 +55,18 @@ a.copybtn {
   margin-left: 6px;
 }
 
-@media (min-width: 1025px) {
+@media (min-width: 1271px) {
     #content {
       margin-left: 0;
       min-height: 728px;
     }
     #content.active {
       margin-left: 315px;
+      margin-right: 315px
```

Comment:
Is this missing semi-colon?

Reply:
that is correct! it is missing semi-colon; however, my testing shows that the semi-colon is optional and it works as expected without semi-colon. I am going to add semi-colon ...